v63004/u_inref/gtm8699.csh

```diff
@@ -11,11 +11,50 @@
 #								#
 #################################################################
 #
+foreach share_opt ("STAT" "NOSTAT")
 
-echo "# Create a single region DB with region DEFAULT"
-$gtm_tst/com/dbcreate.csh mumps >>& dbcreate_log.txt
+	echo "# Testing for $share_opt"
 
-echo '# Run gtm8699.m to run $VIEW and V[IEW]'
-$ydb_dist/mumps -run gtm8699
+	#### testA ####
+	echo ''
+	echo "# Create a single region DB with region DEFAULT"
+	$gtm_tst/com/dbcreate.csh mumps 2 >>& dbcreate_log.txt
 
-$gtm_tst/com/dbcheck.csh >>& dbcreate_log.txt
+	echo '# Setting DB stat settings'
+	$MUPIP set -$share_opt  -reg "*" >>& dbcreate_log.txt
+
+	echo '# Run gtm8699.m'
+	$ydb_dist/mumps -run gtm8699
+
+
+	$gtm_tst/com/dbcheck.csh >>& dbcreate_log.txt
+
+	##### testB ####
+	#echo ''
```

Comment:
I see various commented out lines. Can they be removed if they don't add any value.